go/ephemeral/lib.go

```diff
@@ -385,6 +385,10 @@ func (e *EKLib) BoxLatestUserEK(ctx context.Context, receiverKey libkb.NaclDHKey
 	}, nil
 }
 
+func (e *EKLib) PrepareNewUserEKForPUK(ctx context.Context, merkleRoot libkb.MerkleRoot, pukSigning *libkb.NaclSigningKeyPair) (sig string, boxes []keybase1.UserEkBoxMetadata, newMetadata keybase1.UserEkMetadata, myBox *keybase1.UserEkBoxed, err error) {
```

Comment:
let's rename to `NewUserEKFromPUK` .. slightly less verbose

Reply:
then maybe just `PrepareNewUserEK` -- `ForPUK` is kinda redundant since the args have a PUK signing